src/test/java/hw6/MyJDITest.java

```diff
@@ -0,0 +1,33 @@
+package hw6;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import hw6.entities.MetalsAndColors;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import static hw6.JDITestData.DEFAULT_USER;
+
+public class MyJDITest {
```

Comment:
This is My test, not yours =)
Seriously, this class should be named like this - JdiMetalsAndColorsTest

Reply:
fixed